### PR TITLE
Fix migration issues from last commits

### PR DESCRIPTION
**Pull request by UsamaEquinorAFK**

### CODE REVIEW

File: backend/api/Migrations/20231115100201_inspectionfindingsupdates.cs
Comment (Christdej) [inline backend/api/Migrations/20231115100201_inspectionfindingsupdates.cs:1]: Please use the same naming-convention as we have done for the previous migrations

**Review by Christdej — CHANGES_REQUESTED** [review]

File: backend/api/Migrations/20231115100201_inspectionfindingsupdates.cs
Comment (UsamaEquinorAFK) [reply backend/api/Migrations/20231115100201_inspectionfindingsupdates.cs:1]: Fixing in a different PR

Comment (UsamaEquinorAFK) [reply backend/api/Migrations/20231115100201_inspectionfindingsupdates.cs:1]: https://github.com/equinor/flotilla/pull/1185 fixed hereinstead

**Comment by UsamaEquinorAFK** [comment]
https://github.com/equinor/flotilla/pull/1185 handled by this

**Pull request closed without merging**